server/models/article.js

```diff
@@ -0,0 +1,36 @@
+export default (sequelize, DataTypes) => {
+  const Article = sequelize.define('Article', {
+    title: {
+      allowNull: false,
+      type: DataTypes.STRING,
+    },
+    slug: {
+      allowNull: false,
+      type: DataTypes.STRING,
+    },
+    description: {
+      allowNull: false,
+      type: DataTypes.STRING,
+    },
+    body: {
+      allowNull: false,
+      type: DataTypes.STRING,
+    },
+    favoriteCount: {
+      allowNull: true,
+      type: DataTypes.INTEGER,
+    },
+    categoryId: {
```

Comment:
Having this here without the Category might not be useful. You can take this out. When working on the Categories, we'd do this linkage

Reply:
Ok, thanks.